clients/ts/signalr/spec/HubConnection.spec.ts

```diff
@@ -2,10 +2,11 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 import { HubConnection } from "../src/HubConnection";
+
```

Comment:
You have some new lines in your import statements. Are these intentional/organizational?

Reply:
Not this one 😅 

But yes, I tried to separate: External Packages (`buffer`, etc.), Internal **Package** references (`@aspnet/signalr`, etc.), References to product code (`../src/Module`) and finally "Local" references (`./Module`). It seemed like a good way to break things up.